tests/framework/microvm.py

```diff
@@ -512,8 +512,6 @@ def spawn(
         # and leave 0.2 delay between them.
         if "no-api" not in self.jailer.extra_args:
             self._wait_create()
-        if self.log_file:
-            self.check_log_message("Running Firecracker")
```

Comment:
Hmm, we expect this message to be in the log, to know that Firecracker hasn't crashed. Why do we remove it?

Reply:
hmm making the message consistent sounds good. As for printing that every time we change log target, I would prefer if that message was only printed once at the beginning of FC. When changing the logger config could we instead print "Changed logger config to X".

Reply:
I think there is also the problem that our default level is `Warn` so if its `info!("Running Firecracker")` it won't print.

Reply:
Well in the integration tests we configure the logger with "Debug", so we should be covered. But moving this log message to info sounds like a good idea.

Reply:
The problem is this message happens before the user configuration is applied.

The logger is immediately initialized when starting Firecracker with this PR (before even parsing the command line input), but it is still re-configured with the user command line input at the same point. So it is possible and likely that warnings get printed before the `Running Firecracker` message. To ensure its the first message we need to either ensure no logs or possible errors happen before it (which I don't think is possible) or we set the default level to `info` so it can be put immediately after initialization and still print.

Reply:
> We could create a fake log message using println!() that is formatted just like the logger would format an info!() message using the default configuration

This could work but its a bit messy imo.

I would prefer if we could change the default log level to `Info`. Are there are problems with this?

Reply:
I'm ok with changing it to info. I hope nobody is depending in the exact message like we did. :crossed_fingers: 

On the other hand, I am also OK with this message just being printed in the logs with no formatting since that's how it has always been.

Reply:
I would agree with order 2 -> 1 -> 3.

Reply:
I've implemented approach 2.